config/routes.rb

```diff
@@ -4,7 +4,7 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :teachers do
     resources :teacher_subjects, only: [:create, :destroy]
-    resources :groups
+    resources :study_groups
```

Comment:
here too, why did we move to groups for study_groups? I find it more confusing... and it wasn't changed everywhere, only in these two places I think

Reply:
yes, bcz I dida mistake in the morning. Good catch